frontend/src/pages/staking/overview/StakingOverview.js

```diff
@@ -147,6 +147,7 @@ const StakingOverview = () => {
       let chainnetwork = wallet.networks[chainIds[i]];
       let address = chainnetwork?.walletInfo?.bech32Address;
       let baseURL = chainnetwork?.network?.config.rest;
+      let denom = chainnetwork.network?.config?.currencies[0]?.coinDenom;
```

Comment:
```suggestion
      const denom = chainnetwork.network?.config?.currencies[0]?.coinDenom;
```
Always use const if you are not overriding value